paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionScreen.kt

```diff
@@ -203,14 +228,24 @@ private fun CvcRecollectionButton(isComplete: State<Boolean>, onConfirmPressed:
     }
 }
 
+private fun rememberElement(controller: CvcController): CvcElement {
+    return CvcElement(
+        IdentifierSpec(),
+        controller
+    )
+}
+
 @Composable
 @Preview
 private fun CvcRecollectionFieldPreview() {
     StripeTheme {
         CvcRecollectionScreen(
             cardBrand = CardBrand.Visa,
             lastFour = "4242",
-            isTestMode = true,
+            isTestMode = false,
+            controller = CvcController(
+                cardBrandFlow = stateFlowOf(CardBrand.Visa)
```

Comment:
(not for this PR, but maybe for a follow up: I wonder if we could refactor this so that the card brand isn't a state flow here. I don't think it needs to be a state flow when you're editing the cvc)